tools/src/commands/AndroidShellApp.ts

```diff
@@ -65,5 +65,9 @@ export default (program: any) => {
     .option('--keystoreAlias [string]', 'Keystore alias (optional)')
     .option('--keystorePassword [string]', 'Keystore password (optional)')
     .option('--keyPassword [string]', 'Key password (optional)')
+    .option(
+      '--privateConfigFile [string]',
+      'Path to privateConfig file (aka exp.android.config in app.json) (optional)'
+    )
```

Comment:
yes, it was be used mainly from turtle builder. the code is here: https://github.com/expo/xdl/blob/f7e0b769e8a91d57ccff9e0180760bbc33e2bd45/packages/xdl/src/detach/AndroidShellApp.js#L304

the reason to add this is for `et android-shell-app --url {published_manifest_url}` since `exp.android.config` is not available from published manifest. that is an enhancement to add private api keys support for `et android-shell-app` 
https://github.com/expo/xdl/blob/f7e0b769e8a91d57ccff9e0180760bbc33e2bd45/packages/xdl/src/detach/AndroidShellApp.js#L398-L412